docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst

```diff
@@ -0,0 +1,39 @@
+====================
+Ansible project 2.10
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>` package as well. All dates are subject to change, please see :ref:`base_roadmap_2_10` for the most recent updates on ansible-base.
+
+.. contents::
+   :local:
+
+Release Schedule
+----------------
+
+.. note:: Dates subject to change.
+.. note:: We plan to post weekly alpha releases to the `PyPI ansible project <https://pypi.org/project/ansible/>`_ for testing.
+
+- 2020-06-23: ansible-2.10 alpha freeze.
+  No net new collections will be added to the ``ansible-2.10`` package after this date.
+- 2020-07-10: Ansible collections freeze date for content shuffling.
+  Content should be in its final collection for the ansible-2.10 release.
```

Comment:
```suggestion
  Content should be in its final collection for the ansible-2.10 release, ie no more content to be moved out of ``community.general`` or ``community.network``.
```

Reply:
Thanks @gundalow, I rolled this into 69b44a960b2684a72fa25d2b477e46980fdf03d4 also.